docs/membership-policy.md

```diff
@@ -0,0 +1,41 @@
+# CI maintainers
+
+This document outlines the various responsibilities of the maintainer role in
+KubeVirt CI. It is inspired by the [KubeVirt community membership document] and
+the [Kubernetes community membership document].
+
+KubeVirt CI maintainers are community members that can control CI
+infrastructure.
+
+## Requirements
+
+* Be an [approver] of https://github.com/kubevirt/project-infra for at least 1
+month.
+
+* A nomination by a [current CI maintainer].
```

Comment:
```suggestion
* A reasoned nomination by a [current CI maintainer].
```
Can we state where should the nomination take place? In kubevirt-dev mailing list? Just as a PR to the [current CI maintainer] list?

Reply:
I think it would be better just a PR with the changes to the list, and include the reasoning in the PR description, already pushed PTAL.